db/migrate/20230328103643_add_users_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [ ] Your comments table currently has `user_id`, instead of `author_id`. Kindly fix that. You can look at [this comment](https://stackoverflow.com/a/42056089) on how to fix this.